src/main/java/org/alliancegenome/curation_api/services/mati/MaTIService.java

```diff
@@ -10,24 +10,11 @@
 
 @ApplicationScoped
 public class MaTIService {
-//	@ConfigProperty(name = "okta.client.id") TODO: reading @ConfigProperties is broken inside integ tests
-	private String client_id;
-//	@ConfigProperty(name = "okta.client.secret") TODO: workaround with env vars needs github actions support
-	private  String client_secret;
-//	@ConfigProperty(name = "okta.url")
-	private String okta_url;
-//	@ConfigProperty(name = "okta.scopes")
-	private String okta_scopes;
-//	@ConfigProperty(name = "mati.url")
-	private String mati_url;
-
-	public MaTIService() {
-		client_id = System.getenv("OKTA_CLIENT_ID");
-		client_secret = System.getenv("OKTA_CLIENT_SECRET");
-		okta_url = System.getenv("OKTA_URL");
-		okta_scopes = System.getenv("OKTA_SCOPES");
-		mati_url = System.getenv("MATI_URL") + "api/identifier";
-	}
+	private String client_id = System.getenv("OKTA_CLIENT_ID");
+	private  String client_secret= System.getenv("OKTA_CLIENT_SECRET");
+	private String okta_url = System.getenv("OKTA_URL");
+	private String okta_scopes = System.getenv("OKTA_SCOPES");
+	private String mati_url = System.getenv("MATI_URL") + "api/identifier";
```

Comment:
These variables are already defined in the curation software was the following:
```java
	@ConfigProperty(name = "okta.authentication")
	Instance<Boolean> okta_auth;

	@ConfigProperty(name = "okta.url")
	Instance<String> okta_url;

	@ConfigProperty(name = "okta.client.id")
	Instance<String> client_id;

	@ConfigProperty(name = "okta.client.secret")
	Instance<String> client_secret;

	@ConfigProperty(name = "okta.api.token")
	Instance<String> api_token;
	```

Reply:
Added a temporary controller to be deleted after testing and after using the service to mint DA IDs